docs/layouts/shortcodes/generated/rest_v1_dispatcher.html

```diff
@@ -1277,6 +1277,9 @@
           "parallelism" : {
             "type" : "integer"
           },
+          "slotSharingGroupId" : {
```

Comment:
Do I get it correctly that this is the consequence of [this commit](https://github.com/apache/flink/pull/24380/commits/6dec90667d24b4f197ad9a814f30b99ffe77a6af) not having regenerated the docs? If so, it could be better to have a separate [hotfix] commit in this PR for clarity.

Reply:
yup

Reply:
fixed!